metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/MetadataAuditEventsProcessor.java

```diff
@@ -90,78 +89,25 @@ public void consume(final ConsumerRecord<String, GenericRecord> consumerRecord)
 
     try {
       final MetadataAuditEvent event = EventUtils.avroToPegasusMAE(record);
-      final MetadataAuditOperation operation =
```

Comment:
Do we need to remove deletes from MAE processor? Feel like we can leave this one as is and remove it as a whole when we remove MAE in gms

Reply:
@dexter-mh-lee I think this removal change is still a good idea. Since this code is never going to run, removing it will prevent anyone else going down an unfortunate rabbit hole of debugging

Reply:
Sounds good